bundles/org.openhab.binding.comfoair/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -1587,6 +1606,9 @@
 		<label>Error Current</label>
 		<description>Current errors</description>
 		<category>Text</category>
+		<tags>
+			<tag>Alarm</tag>
```

Comment:
Not sure about `Alarm` here as well (alarm sounds like a siren going off). There is no immediate action required, usually.

Reply:
Ok. I will probably delete it..

Reply:
Ok, maybe adding a tag for `Error` or `Notification` would make sense (also in other bindings), but I'm not even sure if this would better be a property (e.g. in combination with `Status`) or a point tag...